CommandHelper/LocalPackages/commands/buffs/toppo.ms

```diff
@@ -0,0 +1,34 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': '§4T§co§ep§4p§co',
+            'class': 'toppo',
+            'sig': 'rCbYbUC0L7jPINP8S6CNfXD79bAR/iZ5uLQLSWJM0mfN5mqoWm7Z57ZpOa0p6ekS2BiQ3pn08sT5yk7soExfZz05vtFAywgW5se6cP+wo4lQ6cRiLdvCSFNmYsnJkfcJjPzwlOnLR8euSnB9UzcS+Br00YHhLMoOwE4RuzfShEbL6xG89XEcE/7AMboU820PvaKAFDUdV6ymjYlcpG/FiwVw+avPNw15svesqkqniab6zphZvXKcQ9YvcqoNvnV3vVhsegGpcALpLeCQafp23IfLv3t2bQHE8EpkBfkv/4cubNCjEC9SYqQeDLN7x/5LpNw4A8m1xYuNh/9YrYnLxUwmZVBaGw+rqLEdYyAvwiDatTKSd5SzT2vnvMEv7eF5aclvN6sDWv8/28xtueWcuCqPZyiirxDNbgVIb8ho2iaYUvFTtn0wAwmuK55KMcXuxVtIyK+tUDtGcyqKS6+bPr7ITcQqbJn+ArGoL+IPLS6qA7mkB0oZvGOEQdp9sTv6smWR6XsZVNeccMFj62qdTsN742msdPv9QQ29BhOPtZoyFaXVRClhKH/6vob7vd5Q5/NzRNtyGBVWZlbTwgExtfcfx2EnWAZThLCczCNCVGG8gruluCDEQazoewtNKoBYmpQu7+CeUdshLe0rNjhSZkO5Gij+ri34SfgY+WZdE8I='
+            'val': 'eyJ0aW1lc3RhbXAiOjE1MTgyODU5ODU5NDUsInByb2ZpbGVJZCI6IjA4ODBlY2NjZmVlZDRhMmY5ZTIyZDhmNzIxZjM5MTNlIiwicHJvZmlsZU5hbWUiOiJaZUVuZm9yY2VyIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9hYzVlNmY4ZTRjZmQyOGEyNTgxOWE1OWJkYzFjMGYzODRmNWI2MWM5OTk1OTdjZWNiNTY3ZTgxYzMxYzhkZTMifX19'
+            'trail': '',
+            'op': 'false',
+            'speech': "You cannot beat a destrution GOD!!",
+            'powers': array(
+                "/buff",
+                "/jump <0-9>",
+                "/run <0-9>",
+                "/haste <0-9>",
+                "/flyspeed <number>",
+                "/fly",
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/destroyworld",
+                "Anything you attack, dies"
+            ((,
```

Comment:
Now you switched these ones instead of the other ones. delete these two left paranthesis and replace with a right paranthesis